build.gradle

```diff
@@ -439,6 +439,10 @@ project(":api") {
         rename { String fileName ->
             api_spec_path.endsWith(fileName) ? 'openapi.yaml' : fileName  // rename yaml to openapi.yaml
         }
+        copy {
+            from "${projectDir}/candlepin-api-spec.yaml"
+            into "${rootDir}/server/src/main/webapp/docs/"
+        }
```

Comment:
we should also make sure this copying happens on the maven side too (since tito uses maven to build the war)

Reply:
Good point. Made changes to ```api/pom.xml```.

Reply:
You can't add plugins in api/pom.xml, due to the fact we auto-generate the poms with the `gradlew pom` command. Look at the `task pom` in build.gradle to see how to add the plugins during the auto-generation.

Reply:
I was not aware of this. Changes made to ```build.gradle```

Reply:
In addition to making the plugins auto-generate from the gradle pom command, we should also run the command and commit the pom changes as well. (this is basically what we do now when updating any library in build.gradle, we run `./gradlew pom` and commit both the gradle & pom changes

Reply:
Done.